Tests/Serilog.Exceptions.Test/Destructurers/ReflectionBasedDestructurerTest.cs

```diff
@@ -428,66 +432,66 @@ private static Exception ThrowAndCatchException(Action throwingAction)
             }
 
             Assert.True(false, $"{nameof(throwingAction)} did not throw");
-            return null;
+            return null!; // We should never reach this line.
         }
 
         private static ReflectionBasedDestructurer CreateReflectionBasedDestructurer() =>
             new(10);
 
         public class MyObject
         {
-            public string Foo { get; set; }
+            public string? Foo { get; set; }

```

Comment:
Maybe we could add tests for non-nullable references as well?
Just for the sake of completeness.

Reply:
What do you have in mind?

Reply:
It seems that you have changed all properties in all test classes to nullable references.
I thought it would be good to have `MyObject` and `MyObjectWithNonNullable` which would have the same properties just non-nullable.
The purpose is to make sure the API works now and in the future without any issues for both cases.

Reply:
My understanding is that nullable `?` and `!` has no effect on the running of the code or on reflection.